components/Author/Tabs/FeedCard.tsx

```diff
@@ -541,6 +558,26 @@ const styles = StyleSheet.create({
     fontSize: 14,
     marginRight: 5,
   },
+  tag: {
+    display: "inline-block",
+    backgroundColor: "#E8EEFD",
```

Comment:
We try to use only colors defined in `colors.js` btw. I know you are waiting on design for Tetiana so this is just a heads up. Sometimes she uses colors that are slightly off our codebase in which case we figure out whether to use our own or introduce a new color in the palette.